app/controllers/home_controller.rb

```diff
@@ -19,15 +19,17 @@ def dashboard
       user_id: current_user.id,
       recent: true,
       include_private: true,
-      include_adult: !current_user.sfw_filter?
+      include_adult: !current_user.sfw_filter?,
```

Comment:
Yeah, I thought it was strange but the reason I found for that is to make it easier to add more lines and I thought you left it in with that in mind.